ui/src/app/pages/data/cohort/cohort-resource-card.tsx

```diff
@@ -20,6 +20,7 @@ interface Props extends WithConfirmDeleteModalProps, WithErrorModalProps, WithSp
 
 interface State {
   showRenameModal: boolean;
+  referencingDataSets: Array<DataSet>;
```

Comment:
Restores the functionality from https://precisionmedicineinitiative.atlassian.net/browse/RW-2945 which was accidentally removed in a previous refactor